book/src/remapping.md

```diff
@@ -1,29 +1,28 @@
 # Key Remapping
 
-One-way key remapping is temporarily supported via a simple TOML configuration
+Helix currently supports one-way key remapping through a simple TOML configuration
 file. (More powerful solutions such as rebinding via commands will be
 available in the future).
 
-To remap keys, write a `config.toml` file in your `helix` configuration
-directory (default `~/.config/helix` in Linux systems) with a structure like
+To remap keys, create a `config.toml` file in your `Helix` configuration
```

Comment:
```suggestion
To remap keys, create a `config.toml` file in your `helix` configuration
```

since this is referring to the directory name and it will matter on case-sensitive filesystems